### PR TITLE
Put working link for bounding box format

### PR DESCRIPTION
**Pull request by mohantym**
Throwing 404 error for bounding box format hyperlink.
So Put the working link for the bounding box format. 
https://keras.io/api/keras_cv/bounding_box/formats/

### CODE REVIEW

File: guides/ipynb/keras_cv/coco_metrics.ipynb [hunk 1]
```diff
@@ -44,7 +44,7 @@
     "`bounding_box_format` parameter.  This parameter is used to tell the components what\n",
     "format your bounding boxes are in.  While this guide uses the `xyxy` format, a full\n",
     "list of supported formats is available in\n",
-    "[the bounding_box API documentation](/api/keras_cv/bounding_box/formats).\n",
+    "[the bounding_box API documentation](https://keras.io/api/keras_cv/bounding_box/formats/).\n",
```
Comment (fchollet) [inline guides/ipynb/keras_cv/coco_metrics.ipynb:47]: Please:
- Remove the `https://keras.io` prefix
- Apply the change to the `py` and `md` files as well.

Comment (mohantym) [reply guides/ipynb/keras_cv/coco_metrics.ipynb]: Hi @fchollet ! The link is not working . If I remove the keras.io prefix.

Comment (fchollet) [reply guides/ipynb/keras_cv/coco_metrics.ipynb:47]: I guess you're trying to follow the link when hosting the notebook outside of the Keras website, e.g. on Colab?

Comment (mohantym) [reply guides/ipynb/keras_cv/coco_metrics.ipynb:47]: Yeah. You are correct. It is working fine within website but throwing error when opened from  Github source code.

Comment (mohantym) [reply guides/ipynb/keras_cv/coco_metrics.ipynb:47]: Shall i close this PR then?

Comment (fchollet) [reply guides/ipynb/keras_cv/coco_metrics.ipynb:47]: I think it's a legit concern. We can merge the PR. Please also update the `md` file.

**Review by fchollet — APPROVED** [review]

File: guides/ipynb/keras_cv/coco_metrics.ipynb (hunk 1 above)
Comment (fchollet) [reply guides/ipynb/keras_cv/coco_metrics.ipynb:47]: Ok, I see it was updated already. Good to go!

**Pull request merged**